bikeshed/boilerplate/webapps/header.include

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>[TITLE]</title>
+  <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-[STATUS]" rel=stylesheet type="text/css">
```

Comment:
Use the [W3C-STYLESHEET] macro for the whole link instead; the stylesheet urls don't all follow this pattern.

Reply:
I'll fix this one up. Should I send a separate PR for webplatform? (this was just copy/pasta from the webplatform folder)